src/css/wizard-navigation-bar.scss

```diff
@@ -16,6 +16,11 @@ $aw-param-indicator-state: null;
 // wizard step state: 'default', 'current', 'done' etc.
 $aw-param-step-state: null;
 
+@mixin rtl {
+  @at-root [dir="rtl"] #{&} {
+    @content;
+  }
```

Comment:
Out of interest: Do you think it would be a good idea to maybe also add a dedicated:

```scss
@mixin ltr {
  @at-root [dir="ltr"] #{&} {
    @content;
  }
}
```

mixin to the stylesheets, or do you think that this is contraproductive?

Reply:
I think it's unnecessary because you usually build your styles with a default direction in mind (in our case it's LTR) and then add the other direction styles (RTL).
And also by adding an extra LTR mixin you are increasing the generated css styles.

Reply:
Yes you are right. In addition I think it is harder to convey the "default" behaviour of `ltr` if we add a mixin for it.